### PR TITLE
Reviewing project

### PR DESCRIPTION
**Pull request by jpol01**
#Reviewing-trial-project.

>Objective: Understand the code review process by applying what we learnt in the previous lesson.

>**The Code Reviewer won't check your code but the review you performed.**  

> I added two general comments because the system didn't allow me to add the comments exactly in the line that need it.

>There is no mention of the Linters for this project because we are supposed to focus on the evaluation performance. However, I added them to avoid any mistakes.

### CODE REVIEW

**Review by jpol01 — COMMENTED** [review]
Hello, good job. check the tag option and kindly change it back to lowercase. Thanks

**Review by jpol01 — COMMENTED** [review]
Hello, line #63 is working perfectly, however, I recommend you to use a P tag instead of the span tag to follow the semantic guidelines for HTML documents please verify it. Again good job

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @jpol01, 

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [ ] Please consider utilizing the "Review changes" feature from GitHub
- [ ] Please also use GitHub inline comments feature to perform code reviews to your project.
- [ ] Please check out this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) on how to give a good code review 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -12,7 +16,7 @@
     <div class="container">
 
       <!-- PAGE HEADER -->
-      <DIV class="header">
+      <DIV  class="header" >
         <h1 id="title" class="text-center">HAPPY SURVEY</h1>
         <p id="description" class="description text-center">
           Another happy survey, we really want to hear about you!!
```
Comment (jpol01) [inline index.html:22]: Please verify the div tag, should be in lower case to follow a proper semantic. Thanks.

File: index.html [hunk 2]
```diff
@@ -55,12 +59,11 @@ <h1 id="title" class="text-center">HAPPY SURVEY</h1>
...
             class="form-control"
-            placeholder="Your age goes here "
-          />
+            placeholder="Your age goes here "/>
         </div>
 
         <div class="form-group">
-          <SPAN>How do you get your income?</SPAN>
+          <SPAN >How do you get your income? </SPAN>
           <select id="dropdown" name="role" class="form-control" required>
```
Comment (jpol01) [inline index.html:67]: Please change this span tag for a p tag that would be more in line with the semantic HTML uses. Thanks

File: index.html [hunk 3]
```diff
@@ -114,14 +117,14 @@ <h1 id="title" class="text-center">HAPPY SURVEY</h1>
...
             <option value="openSource">Free Books</option>
           </select>
         </div>
 
         <div class="form-group">
-          <DIV>
+          <DIV >
             Please select one of this advantages
             <span class="clue">(Check all that apply)</span>
           </DIV>
```
Comment (jpol01) [inline index.html:130]: Please change the div tag for a p tag in lower case, Thanks for a good job.

**Review by jpol01 — COMMENTED** [review]
Hi @jpol01,
Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

Status: Required Changes ♻️⛔️
Check the comments under the review.

 Please use the **Lower case**, for your tags. 
 Please also check this article about [the semantic of html](https://www.w3schools.com/html/html5_semantic_elements.asp)

**Review by lily-coder — APPROVED** [review]
Hi @jpol01, 

Thank you for implementing the changes as requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**